OrbitGl/App.cpp

```diff
@@ -52,6 +52,7 @@
 ABSL_DECLARE_FLAG(bool, devmode);
 ABSL_DECLARE_FLAG(bool, local);
 ABSL_DECLARE_FLAG(bool, enable_tracepoint_feature);
+ABSL_DECLARE_FLAG(bool, thread_ordering_feature);
```

Comment:
I prefer to call it "track_ordering_feature"

Reply:
Good point, I'll change it.